docs/vector-db-schema-design.md

```diff
@@ -0,0 +1,136 @@
+| `phone_number`                   |           text | `"+380-44-123-4567"`                                              | international\_phone\_number                    |
+| `url`                            |           text | `"https://maps.google.com/..."`                                   | -                                               |
+| `opening_hours`                  |         object | see detailed structure below                                      | structured weekly object (see below)            |
+| `price_level`                    |            int | `2`                                                               | numeric from Google Places (0..4)               |
+| `rating`                         |         number | `4.6`                                                             | float (0.0–5.0)                                 |
+| `reviews`                        |      object\[] | see `reviews` structure below                                     | array of review objects (not just text)         |
+| `tags`                           |        text\[] | `["establishment", "food", "park", "point_of_interest", "store"]` | from `types`                                    |
+| `wheelchair_accessible_entrance` |        boolean | `true`                                                            | from place details                              |
+| `serves_beer`                    |        boolean | `true`                                                            |                                                 |
+| `serves_breakfast`               |        boolean | `false`                                                           |                                                 |
+| `serves_brunch`                  |        boolean | `true`                                                            |                                                 |
+| `serves_dinner`                  |        boolean | `true`                                                            |                                                 |
+| `serves_lunch`                   |        boolean | `true`                                                            |                                                 |
+| `serves_vegetarian_food`         |        boolean | `true`                                                            |                                                 |
+| `serves_wine`                    |        boolean | `true`                                                            |                                                 |
+| `takeout`                        |        boolean | `true`                                                            |                                                 |
+| `last_updated`                   |       dateTime | `"2025-08-20T22:05:00+03:00"`                                     | when the DB record was last updated             |
```

Comment:
Properties like `tags` and `administrative_area_level_1/2` may benefit from referencing a controlled vocabulary or enum for consistency.